index.js

```diff
@@ -116,7 +116,11 @@ class CreateCertificatePlugin {
    * Creates a certificate for the given options set in serverless.yml under custom->customCertificate
    */
   createCertificate() {
-
+    const region = this.options.region ? this.options.region : this.serverless.service.provider.region;
+    if (region.startsWith('cn-')) {
+      this.serverless.cli.log('Skipping certificate summary in Chinese region');
```

Comment:
this log msg does not fit what the code is doing

Reply:
Can you elaborate?

Reply:
the log you added here says - skipping certificate summary. But it is actually skipping the creation of the certificate all together

Reply:
You're right. Thanks! :)